compiler/one-cmds/one-prepare-venv

```diff
@@ -35,6 +35,7 @@ fi
 
 VER_TENSORFLOW=2.6.0
 VER_ONNX=1.10.1
+VER_ONNXRUNTIME=1.10.0
```

Comment:
Is this available in both Ubuntu 18.04 and 20.04 ?

Reply:
Checked, everything works.

P.s. to satisfy my curiosity, tried to install it without venv: 20.04 - ok, 18.04 - fail, highest available version is 1.3.0